test/bellecour-fork/voucher-config.js

```diff
@@ -1 +1 @@
-export const VOUCHER_HUB_ADDRESS = 'TO BE DEPLOYED';
+export const VOUCHER_HUB_ADDRESS = '0xfaAddC93baf78e89DCf37bA67943E1bE8F37Bb8c';
```

Comment:
Is it okay to commit this or is it better to keep it to 'TO BE DEPLOYED'?
Or is it the definitive voucher hub address?

Reply:
the address should remain 'TO BE DEPLOYED' until the voucher is deployed on prod. it was just an error on my part, i accidentally committed the address I use for local tests. 

correction commit : 6134aac5a8a2e23537e2e0f58c8ba43e55c7356a